pkg/idp/endpoint_test.go

```diff
@@ -105,3 +105,93 @@ func TestDeleteIdentityProviderEndpoint(t *testing.T) {
 	_, err := e(ctx, req)
 	assert.Nil(t, err)
 }
+
+func TestGetIdentityProviderMappersEndpoint(t *testing.T) {
+	var mockCtrl = gomock.NewController(t)
+	defer mockCtrl.Finish()
+
+	var mockIdpComponent = mock.NewComponent(mockCtrl)
+
+	var e = MakeGetIdentityProviderMappersEndpoint(mockIdpComponent)
+
+	var realm = "test-community"
+	var ctx = context.Background()
+
+	var req = make(map[string]string)
+	req[prmRealm] = realm
+	req[prmProvider] = idpAlias
+
+	mockIdpComponent.EXPECT().GetIdentityProviderMappers(ctx, realm, idpAlias).Return([]api.IdentityProviderMapperRepresentation{}, nil)
+	_, err := e(ctx, req)
+	assert.Nil(t, err)
+}
+
+func TestCreateIdentityProviderMapperEndpoint(t *testing.T) {
+	var mockCtrl = gomock.NewController(t)
+	defer mockCtrl.Finish()
+
+	var mockIdpComponent = mock.NewComponent(mockCtrl)
+
+	var e = MakeCreateIdentityProviderMapperEndpoint(mockIdpComponent)
+
+	var realm = "test-community"
+	var ctx = context.Background()
+
+	var req = make(map[string]string)
+	req[prmRealm] = realm
+	req[prmProvider] = idpAlias
+
+	mapper := createTestApiIdpMapper()
+	mapperJSON, _ := json.Marshal(mapper)
+	req[reqBody] = string(mapperJSON)
+
+	mockIdpComponent.EXPECT().CreateIdentityProviderMapper(ctx, realm, idpAlias, mapper).Return(nil)
+	_, err := e(ctx, req)
+	assert.Nil(t, err)
+}
+
+func TestUpdateIdentityProviderMapperEndpoint(t *testing.T) {
```

Comment:
Update should contains at least 3 tests... (3 return statements)
* Missing/Invalid JSON
* Invalid body (`mapper.Validate()` fails)
* Call to authorization component